prow/config.yaml

```diff
@@ -955,6 +955,68 @@ presubmits:
     branches:
     - release-1.6
 
+  - name: pull-kubernetes-e2e-gce-ipvs
+    agent: kubernetes
+    skip_branches:
+    - release-1.6
```

Comment:
please add a comment next to these explaining why for anyone reading this (presumably:  `# IPVS wasn't available before 1.8`)

Reply:
Sure, thanks.